src/Microsoft.Diagnostics.Monitoring.Options/EgressOptions.cs

```diff
@@ -11,13 +10,11 @@ namespace Microsoft.Diagnostics.Tools.Monitor
     internal sealed class EgressOptions
     {
         [Display(
-            ResourceType = typeof(OptionsDisplayStrings),
-            Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_EgressOptions_FileSystem))]
+            Description = "Mapping of file system egress provider names to their options.")]
```

Comment:
Was there a reason why you in-lined these? Is it just to be consistent with the other extensions not reaching into the main OptionsDisplayStrings resx?

Reply:
I thought that I'd move EgressOptions out of the Microsoft.Diagnostics.Monitoring.Options assembly and since it no longer is used by product code, it doesn't need localized descriptions any more. I'll take another look at actually moving the class if possible.

Reply:
I'll revert the display string removal and leave moving the class out to another PR.